database/factories/ModelFactory.php

```diff
@@ -95,3 +96,14 @@
 $factory->defineAs(User::class, 'admin', function () use ($factory) {
     return array_merge($factory->raw(User::class), ['role' => 'admin']);
 });
+
+// Campaign Factory
+$factory->define(Campaign::class, function (Generator $faker) {
+
+    return [
+        'id' => $faker->randomNumber,
+        'internal_title' => $faker->sentence(),
```

Comment:
can't wait to see what it comes up with for these

Reply:
Oh man, wouldn't it be fun to make a Markov chain generator based on real campaign names? 😂